client/src/actions/types.js

```diff
@@ -0,0 +1,6 @@
+export const SET_CURRENT_USER = 'SET_CURRENT_USER';
```

Comment:
You should've used `keymirror` package for type definitions, you can take a look at how it's used at it's `npm/github` page.